Jenkinsfile.release

```diff
@@ -11,7 +11,7 @@ CQZ_VERSION = ""
 BUILD_STATUS_FAILED = 'Failed'
 BUILD_STATUS_SUCCESSFUL = 'Successful'
 
-EMAIL_LIST = 'alexander@cliqz.com,leonid@cliqz.com,lucian@cliqz.com'
+EMAIL_LIST = 'alexander@cliqz.com,lucian@cliqz.com'
 EMAIL_LIST_SUCCESSFUL = 'kateryna@cliqz.com,prashant@cliqz.com,tamara@cliqz.com,marta@cliqz.com,oleksandra@cliqz.com'
```

Comment:
Same as above comment here.

Reply:
yes, Marty already suggest this, will do soon